corehq/util/django_migrations.py

```diff
@@ -115,42 +117,55 @@ def noop_migration():
     return RunPython(RunPython.noop, RunPython.noop)
 
 
-def run_once_off_migration(command_name, *args, required_commit=None, **kwargs):
-    """Return a migration operation that can be used to run a management command from
-    a Django migration. This will give the user directions for running the command
-    manually if the command fails or has since been removed from the codebase"""
-    @skip_on_fresh_install
-    def _run_command(apps, schema_editor):
-        run_management_command_or_exit(command_name, required_commit=required_commit, *args, **kwargs)
+def prompt_for_historical_migration(app_name, migration_name, required_commit, *args, **kwargs):
```

Comment:
Additional args / kwargs aren't used:

```suggestion
def prompt_for_historical_migration(app_name, migration_name, required_commit):
```